apps/celview/CMakeLists.txt

```diff
@@ -2,5 +2,10 @@ add_executable(celview
     main.cpp
 )
 
-target_link_libraries(celview NuklearMisc SDL2::SDL2main)
+target_link_libraries(celview NuklearMisc)
+if (MINGW)
+target_link_libraries(celview -lmingw32 SDL2::SDL2main)
```

Comment:
Is this only needed for celview?

Reply:
Also for launcher. Everything else doesn't include SDL2main

Reply:
Hopefully it will be resolved upstream eventually then fix can be removed.